### DISCUSSION TITLE
GitHub Mobile - Select Username to View Profile Everywhere

### DISCUSSION THREAD

**Discussion by achhina** [community/community#139542] [Mobile]
Selecting a username to view profile works on some views not others, i.e. works when viewing commits but not when viewing comments in issues or PRs.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.179.0 (142164528)
Device: iPhone 12 Pro Max (iOS 17.6.1)
</details>

**Comment by stefisha** [comment]
Try updating the app, if you havent already. If not try clearing the app cache, that might remove temporary glitches.
If the issue still persists try reinstalling the app.
Also would advise to try different platforms (like Android) to see if the issue persists.
Hope this helps

**Reply by achhina** [reply]
I hadn't realized this was a glitch. I tried clearing the app cache and deleting and re-downloading the app, unfortunately neither worked.

I also notice that clicking on dates does not show the absolute time, and in another issue one of the comments in the thread mentioned that this should work for iOS. I'm guessing this is related?
